lib/rules/font-family-name-quotes/index.js

```diff
@@ -69,28 +128,26 @@ const rule = (primary) => {
 		}
 
 		root.walkDecls(/^font(-family)?$/i, (decl) => {
-			const fontFamilies = findFontFamily(decl.value);
+			let fontFamiliess = MakeMutableFontFamilies(findFontFamily(decl.value), decl);
 
-			if (fontFamilies.length === 0) {
+			if (fontFamiliess.length === 0) {
 				return;
 			}
 
-			for (const fontFamilyNode of fontFamilies) {
-				let rawFamily = fontFamilyNode.value;
-
-				if ('quote' in fontFamilyNode) {
-					rawFamily = fontFamilyNode.quote + rawFamily + fontFamilyNode.quote;
-				}
-
-				checkFamilyName(rawFamily, decl);
+			for (const fontFamilyNode of fontFamiliess) {
+				checkFamilyName(fontFamilyNode, decl);
 			}
 		});
 
 		/**
-		 * @param {string} rawFamily
-		 * @param {import('postcss').Declaration} decl
+		 * @param { MutableNode } fontFamilyNode
+		 * @param { import('postcss').Declaration} decl
```

Comment:
[suggest] How about removing extra spaces?

```suggestion
		 * @param {MutableNode} fontFamilyNode
		 * @param {import('postcss').Declaration} decl
```